packaging/portmidi.pc.in

```diff
@@ -6,5 +6,5 @@ Name: portmidi
 Description: PortMidi is a cross platform library for accessing operating systems' MIDI APIs
 Version: @CMAKE_PROJECT_VERSION@
 URL: https://github.com/mixxxdj/portmidi
-Libs: -L${libdir} -lportmidi -lporttime
+Libs: -L${libdir} -lportmidi -lporttime @SNDIO_LINK@
```

Comment:
```suggestion
Libs: -L${libdir} -lportmidi -lporttime @PKGCONFIG_LDFLAGS@
```
Let's make this name more generic in case there are some other uses for it in the future.

Reply:
Come to think of it, the pkgconfig file should probably list alsa as a dependency. I'll take care of that in a different PR.